giphy_cli.py

```diff
@@ -1,31 +1,21 @@
-import click
-import requests
-
-API_KEY = "tFRkU1RvdbGFdp76FKKBuHvMRPpCrmM7"  # Use your API key
```

Comment:
So your API key will be permanently part of your history now, so you'll have to remove it. I sent you a link to how to do that in the previous pull request, but it's essential that you fix this.